lib/solargraph/source_map/mapper.rb

```diff
@@ -107,7 +107,8 @@ def process_directive source_position, comment_position, directive
           begin
             src = Solargraph::Source.load_string("def #{directive.tag.name};end", @source.filename)
             region = Parser::Region.new(source: src, closure: namespace)
-            gen_pin = Parser.process_node(src.node, region).first.last
```

Comment:
In the case where a symbol was the default value of method parameter, this was returning the symbol pin, not the method pin